diff --git a/node_modules/@mantine/core/cjs/Accordion/AccordionControl/AccordionControl.js b/node_modules/@mantine/core/cjs/Accordion/AccordionControl/AccordionControl.js
index e4d8206..c338fca 100644
--- a/node_modules/@mantine/core/cjs/Accordion/AccordionControl/AccordionControl.js
+++ b/node_modules/@mantine/core/cjs/Accordion/AccordionControl/AccordionControl.js
@@ -8,7 +8,6 @@ var utils = require('@mantine/utils');
 var Accordion_context = require('../Accordion.context.js');
 var AccordionItem_context = require('../AccordionItem.context.js');
 var AccordionControl_styles = require('./AccordionControl.styles.js');
-var UnstyledButton = require('../../UnstyledButton/UnstyledButton.js');
 
 function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'default' in e ? e['default'] : e; }
 
@@ -76,7 +75,7 @@ const AccordionControl = React.forwardRef((_a, ref) => {
   const isActive = ctx.isItemActive(value);
   const shouldWrapWithHeading = typeof ctx.order === "number";
   const Heading = `h${ctx.order}`;
-  const content = /* @__PURE__ */ React__default.createElement(UnstyledButton.UnstyledButton, __spreadProps(__spreadValues({}, others), {
+  const content = /* @__PURE__ */ React__default.createElement("div", __spreadProps(__spreadValues({}, others), {
     ref,
     "data-accordion-control": true,
     disabled,
@@ -85,7 +84,6 @@ const AccordionControl = React.forwardRef((_a, ref) => {
       onClick == null ? void 0 : onClick(event);
       ctx.onChange(value);
     },
-    type: "button",
     "data-active": isActive || void 0,
     "aria-expanded": isActive,
     "aria-controls": ctx.getRegionId(value),
diff --git a/node_modules/@mantine/core/esm/Accordion/AccordionControl/AccordionControl.js b/node_modules/@mantine/core/esm/Accordion/AccordionControl/AccordionControl.js
index 1ab9e8e..d8981ed 100644
--- a/node_modules/@mantine/core/esm/Accordion/AccordionControl/AccordionControl.js
+++ b/node_modules/@mantine/core/esm/Accordion/AccordionControl/AccordionControl.js
@@ -4,7 +4,6 @@ import { createScopedKeydownHandler } from '@mantine/utils';
 import { useAccordionContext } from '../Accordion.context.js';
 import { useAccordionItemContext } from '../AccordionItem.context.js';
 import useStyles from './AccordionControl.styles.js';
-import { UnstyledButton } from '../../UnstyledButton/UnstyledButton.js';
 
 var __defProp = Object.defineProperty;
 var __defProps = Object.defineProperties;
@@ -68,7 +67,7 @@ const AccordionControl = forwardRef((_a, ref) => {
   const isActive = ctx.isItemActive(value);
   const shouldWrapWithHeading = typeof ctx.order === "number";
   const Heading = `h${ctx.order}`;
-  const content = /* @__PURE__ */ React.createElement(UnstyledButton, __spreadProps(__spreadValues({}, others), {
+  const content = /* @__PURE__ */ React.createElement("div", __spreadProps(__spreadValues({}, others), {
     ref,
     "data-accordion-control": true,
     disabled,
@@ -77,7 +76,6 @@ const AccordionControl = forwardRef((_a, ref) => {
       onClick == null ? void 0 : onClick(event);
       ctx.onChange(value);
     },
-    type: "button",
     "data-active": isActive || void 0,
     "aria-expanded": isActive,
     "aria-controls": ctx.getRegionId(value),
